lazer/sdk/rust/protocol/src/subscription.rs

```diff
@@ -33,6 +34,17 @@ pub struct UnsubscribeRequest {
     pub subscription_id: SubscriptionId,
 }
 
+/// A request to update an existing subscription by adding or removing price feeds and properties.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct UpdateSubscriptionRequest {
+    pub subscription_id: SubscriptionId,
+    pub add_price_feed_ids: Vec<PriceFeedId>,
+    pub remove_price_feed_ids: Vec<PriceFeedId>,
+    pub add_properties: Vec<PriceFeedProperty>,
+    pub remove_properties: Vec<PriceFeedProperty>,
```

Comment:
I can understand updating price feeds. But, is subscription ID and properties something we want to allow updates for? I'm actually not even sure what these properties even are...